### DISCUSSION TITLE
jsub vs submit

### DISCUSSION THREAD

**Discussion by EnsonoStevenPerva** [IBM/zoau-samples#25] [Q&A] [answered]
Hoping someone can provide some clarity as I try to write some documentation to teach some colleagues about ZOAU. Is there a major difference between `jsub` and the `submit` command provided natively by USS? It seems like `jsub` allows me to do some more interesting stuff like seeing verbose output, debug output, and serve as a recipient for other commands. Is there anything, beyond those points, that make `jsub` and `submit` different?

**Comment by MikeFultonDev** [comment, marked as answer]
No - you're right. jsub and submit are pretty darn similar.
